values/src/main/proto/gcp/storage/bucket.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package gcp.storage;
+
+option java_generate_equals_and_hash = true;
+option java_multiple_files = true;
+option java_package = "org.spine3.gcp.storage";
+option java_outer_classname = "BucketProto";
+
+// Google Cloud Datastore bucket
```

Comment:
It would be better to place these files into GCP-related repository, not in the `core-java`.